app/controllers/regular_events_controller.rb

```diff
@@ -40,7 +40,7 @@ def update
     set_wip
     if @regular_event.update(regular_event_params)
       update_publised_at
-      Newspaper.publish(:regular_event_update, @regular_event)
+      Newspaper.publish(:regular_event_update, { regular_event: @regular_event })
```

Comment:
`Newspaper.publish(:event_create, { event: @regular_event })` と書き方が微妙に異なるのが少し気になりました🙇‍♂️
サブスクライブするデータ自体は「イベント」だと思うので、下記の書き方でも不整合はないのではないかと思っています🙇‍♂️
```suggestion
      Newspaper.publish(:regular_event_update, { event: @regular_event })
```

Reply:
コメントありがとうございます！
たしかに`Newspaper.publish(:event_create, { event: @regular_event })`と記載が微妙に異なっていますね＞＜

ただ、以下の理由から個人的にはあまり違和感は覚えませんでした🙌

- `Newspaper.publish(:event_create, { event: @regular_event })`は特別・定期どちらの「イベント」でも使用されるもののため、ハッシュのキーが`event`になっている
- 一方で、`Newspaper.publish(:regular_event_update, { regular_event: @regular_event })`は**定期イベントでのみ**使用されるため、`regular_event `

=>使用されるシーンが「イベント」全般なのか、定期イベントのみなのか、で異なるため記述が違っている

また仮にハッシュのキーを`:event`に変更し、`{ event: @regular_event }`にした場合、
`app/models/regular_event_update_notifier.rb`内も記載を統一するためには
ハッシュを代入する変数名も`regular_event`から`event`に変更が必要になる気がします🤔  
`regular_event_update_notifier.rb`は特別イベントに特化した通知処理を書いたファイルなので
`regular_event`という記載は残したいかな〜と思っております🙌

https://github.com/fjordllc/bootcamp/blob/0a77155568e85bc829a9cd7478d91b8e922cb53b/app/models/regular_event_update_notifier.rb#L3-L12

上記の点について「もっといいやり方がありそう」等ありましたらご教示いただけますと幸いです🙇‍♀️
たびたびお手数ですがご確認の程よろしくお願いいたします🍀

Reply:
> 使用されるシーンが「イベント」全般なのか、定期イベントのみなのか、で異なるため記述が違っている

確かに「イベント」全般という意味で `event` と捉えるとその通りだと思いました👀現状のままでいいと思います！